client/src/holdings/AddTradeDialog.tsx

```diff
@@ -149,10 +139,11 @@ export function AddTradeDialog({
                 label='Date'
                 control={control}
                 render={({onChange, value}) => (
-                  <DateTimePicker
+                  <KeyboardDateTimePicker
                     value={value}
                     onChange={(date) => onChange(date)}
                     variant='inline'
+                    format='dd-MM-yyyy HH:mm:ss'
```

Comment:
Cool that looks better, though I reckon it should be dd/MM/yyyy HH:mm:ss as the dashes are generally used when its yyyy-MM-dd format.

Also is there a way to delay the validation by a second or so, or to when the user stops typing, so that an error doesn't show up straight away?

Reply:
Even managed to get the import sort formatted correctly!